channels_redis/pubsub.py

```diff
@@ -8,7 +9,51 @@
 logger = logging.getLogger(__name__)
 
 
+def _wrap_close(proxy, loop):
+    original_impl = loop.close
+
+    def _wrapper(self, *args, **kwargs):
+        if not self.is_closed():
+            self.run_until_complete(proxy._del_layer(self))
+        self.close = original_impl
+        return self.close(*args, **kwargs)
+
+    loop.close = types.MethodType(_wrapper, loop)
+
+
 class RedisPubSubChannelLayer:
+    def __init__(self, *args, **kwargs) -> None:
+        self._args = args
+        self._kwargs = kwargs
+        self._layers = {}
+
+    def __getattr__(self, name):
+        try:
+            loop = asyncio.get_event_loop()
```

Comment:
get_event_loop() is deprecated in Python 3.10.

Reply:
Thanks - resolved in latest.